css/templates/lune/components/_header.css

```diff
@@ -0,0 +1,51 @@
+@import "../page-vars";
+
+.lune .ampstart-headerbar {
+  display: flex;
+  align-items: center;
+
+  height: var(--layout-header-height-m);
+
+  box-shadow: none;
+
+  will-change: transform;
```

Comment:
The `box-shadow` property overrides the default for `ampstart-headerbar`. 
As the header is fixed to the top of the viewport, the `will-change: transform` forces it onto a new GPU layer. I will add comments to the code to explain.